freud/environment.pyx

```diff
@@ -671,33 +676,34 @@ cdef class MatchEnv(Compute):
 
         """
         points = freud.common.convert_array(points, shape=(None, 3))
-        refPoints = freud.common.convert_array(refPoints, shape=(None, 3))
+        ref_points = freud.common.convert_array(ref_points, shape=(None, 3))
 
         cdef np.ndarray[float, ndim=1] l_points = np.ascontiguousarray(
             points.flatten())
-        cdef np.ndarray[float, ndim=1] l_refPoints = np.ascontiguousarray(
-            refPoints.flatten())
+        cdef np.ndarray[float, ndim=1] l_ref_points = np.ascontiguousarray(
+            ref_points.flatten())
         cdef unsigned int nP = l_points.shape[0]
-        cdef unsigned int nRef = l_refPoints.shape[0]
+        cdef unsigned int nRef = l_ref_points.shape[0]
 
         defaulted_nlist = freud.locality.make_default_nlist_nn(
-            self.m_box, points, points, self.num_neigh, nlist, None, self.rmax)
+            self.m_box, points, points, self.num_neigh, nlist, True, self.rmax)
         cdef freud.locality.NeighborList nlist_ = defaulted_nlist[0]
 
         cdef vector[float] min_rmsd_vec = self.thisptr.minRMSDMotif(
             nlist_.get_ptr(), <vec3[float]*> &l_points[0], nP,
-            <vec3[float]*> &l_refPoints[0], nRef, registration)
+            <vec3[float]*> &l_ref_points[0], nRef, registration)
 
         return min_rmsd_vec
 
-    def isSimilar(self, refPoints1, refPoints2, threshold, registration=False):
-        R"""Test if the motif provided by refPoints1 is similar to the motif
-        provided by refPoints2.
+    def isSimilar(self, ref_points1, ref_points2,
+                  threshold, registration=False):
+        R"""Test if the motif provided by ref_points1 is similar to the motif
+        provided by ref_points2.
 
         Args:
-            refPoints1 ((:math:`N_{particles}`, 3) :class:`numpy.ndarray`):
+            ref_points1 ((:math:`N_{particles}`, 3) :class:`numpy.ndarray`):
```

Comment:
To bring this in line with the rest of freud, can we just rename these to points and ref_points, rather than ref_points[1|2]?